### PR TITLE
Russian Peasant in Javascript

### PR DESCRIPTION
**Pull request by ritvij14**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?
Yes
(Write your answer here.)

### Description
This is the Russian Peasant Algorithm in Javascript, an algorithm to simplify the multiplication of 2 large numbers.
(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
Issue #978 
(Write your answer here.)

### CODE REVIEW

**Comment by ritvij14** [comment]
@iamrajiv @atarax665 @siddharth25pandey @plazzy99 please review this.

File: JavaScript/README.md [hunk 1]
```diff
@@ -9,6 +9,9 @@
 
 # Codes in the Javascript language
 
+## Algorithms
+- [Russian Peasant Algorithm](Algorithms/RussianPeasant.js)
+
```
Comment (iamrajiv) [inline JavaScript/README.md]: remove this line break

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@ritvij14 Format the code using Prettier.

File: JavaScript/README.md (hunk 1 above)
Comment (ritvij14) [reply JavaScript/README.md]: @iamrajiv done.

**Comment by ritvij14** [comment]
@iamrajiv @plazzy99 please review this PR. I have completed the changes.

**Review by vatsalkesarwani12 — APPROVED** [review]
👍🏻

**Review by iamrajiv — APPROVED** [review]

**Comment by ritvij14** [comment]
@atarax665 @siddharth25pandey please review this, only one review is left to merge.

**Pull request merged**